it/controllers/agentAppointment/RatingListOptionsISpec.scala

```diff
@@ -2,23 +2,28 @@ package controllers.agentAppointment
 
 import base.{HtmlComponentHelpers, ISpecBase}
 import com.github.tomakehurst.wiremock.client.WireMock.{aResponse, get, post, stubFor}
+import models.propertyrepresentation.{AgentSelected, SearchedAgent, SelectedAgent}
```

Comment:
Same comments for this spec also
- Can't see any fail scenarios
- Error text content not used in eng&welsh